Arena/ArenaHooks.cs

```diff
@@ -103,8 +104,22 @@ private void ArenaHooks()
             On.MoreSlugcats.SingularityBomb.ctor += SingularityBomb_ctor;
             IL.Player.ClassMechanicsSaint += Player_ClassMechanicsSaint1;
             IL.Player.Collide += (il) => Player_Collide2(il, typeof(Player).GetMethod(nameof(Player.Collide)));
+            new Hook(typeof(Player).GetProperty("CanPutSlugToBack").GetGetMethod(), this.CanPutSlugToBack);
+        }
+        private bool CanPutSlugToBack(Func<Player, bool> orig, Player self)
+        {
+            if (isArenaMode(out var _))

```

Comment:
This is fine for all gamemodes.
```suggestion
            if (OnlineManager.lobby != null)
```

 I also think it would be nice if the players pressing up allowed piggy backing stunned/dead slugcats.

```suggestion
            if (OnlineManager.lobby != null && (self.input[0].y <= 0))
```